namerd/core/src/main/scala/io/buoyant/namerd/package.scala

```diff
@@ -9,4 +11,16 @@ package object namerd {
     /** A Future representing the first non-pending value of this Activity */
     def toFuture: Future[T] = activity.values.toFuture.flatMap(Future.const)
   }
+
+  val Hash = Buf.Utf8("#")
+  val Dollar = Buf.Utf8("$")
+  val NotHashOrDollar: Buf => Boolean =
+    (b: Buf) => !(b == Hash || b == Dollar)
+  val Percent = Path.Utf8("%")
+
+  def stripTransformerPrefix(p: Path): Path =
+    if (p.startsWith(Percent))
+      Path(p.elems.dropWhile(NotHashOrDollar): _*)
+    else
+      p
```

Comment:
I think I'd prefer that we scope this into a `Paths` object and reference as e.g.  `Paths.Percent`, `Paths.stripTransfomerPrefix()?

Also perhaps better to call `Hash`->`ConfiguredNamerPrefix`, `Dollar`->`LoadedNamerPrefix`, `Percent`->`TransformerPrefix`, etc?